appcues/src/main/java/com/appcues/AppcuesFirebaseMessagingService.kt

```diff
@@ -78,10 +78,12 @@ public class AppcuesFirebaseMessagingService : FirebaseMessagingService() {
 
     @JsonClass(generateAdapter = true)
     internal data class AppcuesMessagingData(
-        @Json(name = "notification_id")
-        val id: String,
         val title: String,
         val body: String,
+        @Json(name = "notification_id")
+        val notificationId: String,
+        @Json(name = "notification_version")
+        val notificationVersion: Int? = null,
```

Comment:
was testing some things and discovered this, and L98 below need to be `Long?` actually - they are timestamp values

Reply:
yes, confirmed